store/postgres/src/block_range.rs

```diff
@@ -167,20 +167,26 @@ impl<'a> BlockRangeColumn<'a> {
 }
 
 impl<'a> BlockRangeColumn<'a> {
-    /// Output SQL that matches only rows whose block range contains `block`
-    pub fn contains(&self, out: &mut AstPass<Pg>) -> QueryResult<()> {
+    /// Output SQL that matches only rows whose block range contains `block`.
+    ///
+    /// `filters_by_id` has no impact on correctness. It is a heuristic to determine
+    /// whether the brin index should be used. If `true`, the brin index is not used.
+    pub fn contains(&self, out: &mut AstPass<Pg>, filters_by_id: bool) -> QueryResult<()> {
```

Comment:
Maybe `skip_brin_index` would be a better name than `filters_by_id` in this function, but it's not a big deal.